src/__experimental__/components/decimal/decimal.component.js

```diff
@@ -138,9 +138,21 @@ class Decimal extends React.Component {
   };
 
   removeDelimiters = (value) => {
-    const format = I18nHelper.format();
-    const delimiter = `\\${format.delimiter}`;
-    const delimiterMatcher = new RegExp(`[${delimiter}]*`, "g");
+    /* If a value is passed in that is a number but has too many delimiters in succession, we want to handle this
```

Comment:
From 141 to 152 isn't required as this block of code was moved to the `toStandardDecimal` function. Here is the link to where it is now.
https://github.com/Sage/carbon/pull/3799/files#diff-41d1ef079632dd8b6d4e6270f83eb09c1dcfc8017b6ee137810ea9d3d1f41dc2R200